content/collections/source-catalog/en/snowflake.md

```diff
@@ -23,120 +23,148 @@ exclude_from_sitemap: false
 
-4. Copy the autogenerated SQL query and run it in Snowflake to give Amplitude the proper permissions. 
-5. After running the query, click *Next* to test the connection.
-6. After the test is successful, click *Next* again to move on to the data selection stage.
-7. Choose the modeling method, either [Change Data Capture](#table-selection-ui-settings) or [Custom SQL Query](#custom-sql-query-settings).
+- **Change Data Capture (CDC) Continuous Sync**: Choose this option to directly mirror the data in Snowflake with `INSERT`, `UPDATE`, and `DELETE` operations based on changes detected by Snowflake's CDC feature. This method disables Amplitude's enrichment services to remain in sync with your source of truth and is ideal when you need to keep Amplitude data fully synchronized with your Snowflake data. `UPDATE` and `DELETE` operations mutate data in Amplitude.
 
-### Change Data Capture settings
+{{partial:partials/data/snowflake-strat-comp}}
 
-Configure the modeling method:
+## Prerequisites and considerations for CDC
 
-- **Data source**: Choose a table or view from the left panel.
-- **Data type**: Select if the table maps to event, user property, or group property data.
-- **Frequency**: Select the interval with which Amplitude should check for changes in the Snowflake table.
+When using CDC Continuous Sync, be aware of the following limitations:
 
-Map the required and custom fields: Setup name mapping between columns in the Snowflake data source and data field name that Amplitude requires. For more information, see [Data fields](#data-fields) below.
+- **Enable Change Tracking**: Enable change tracking for the source table or view. See [Enabling Change Tracking on Views and Underlying Tables](https://docs.snowflake.com/en/user-guide/streams-manage.html#label-enabling-change-tracking-views) in Snowflake's documentation.
 
-When complete, click **Test Mapping** to verify the correct data appears under the right property in Amplitude.
+- **Data Retention Settings**: `DATA_RETENTION_TIME_IN_DAYS` must be greater than or equal to `1`, but Amplitude recommends at least `7` days. Otherwise, the change-based import fails. For more details, see [Time Travel](https://docs.snowflake.com/en/user-guide/data-time-travel) in Snowflake's documentation. Setting `DATA_RETENTION_TIME_IN_DAYS` to `0` disables the change tracking and causes the connection to become unrecoverable. If this happens, recreate the source.
 
-### Custom SQL query settings
+- **Disable Change Tracking**: If you disable change tracking in Snowflake or disconnect the Amplitude source for a period longer than the value of `DATA_RETENTION_TIME_IN_DAYS`, Amplitude loses the ability to track historical changes. In this case, recreate the connection. To avoid duplicate events, ensure all events have an `insert_id` set, and recreate the connection within seven days.
```

Comment:
```suggestion
- **Disable Change Tracking**: If you disable change tracking in Snowflake, or disconnect the Amplitude source for a period longer than the value of `DATA_RETENTION_TIME_IN_DAYS`, Amplitude loses the ability to track historical changes. In this case, recreate the connection. To avoid duplicate events, ensure all events have an `insert_id` set, and recreate the connection within seven days.
```